purchasing/conductor/views.py

```diff
@@ -267,3 +271,32 @@ def assign(contract_id, flow_id, user_id):
     db.session.commit()
     flash('Successfully assigned to {}!'.format(user.email), 'alert-success')
     return redirect(url_for('conductor.index'))
+
+@blueprint.route('/upload_new', methods=['GET', 'POST'])
+def upload():
+    form = FileUpload()
+    if form.validate_on_submit():
+        _file = request.files.get('upload')
+        filename = secure_filename(_file.filename)
+        filepath = os.path.join(current_app.config.get('UPLOAD_FOLDER'), filename)
+        _file.save(filepath)
+        main(filepath, filename, None, None, None)
+        return render_template('conductor/upload_success.html', filepath=filepath)
+    else:
+        return render_template('conductor/upload_new.html', form=form)
+
+@blueprint.route('/_process_file', methods=['POST'])
```

Comment:
Yeah I'm getting a reference error on the ajax stuff 
<img width="1228" alt="screen shot 2015-07-06 at 12 55 32 pm" src="https://cloud.githubusercontent.com/assets/3640800/8531753/70a18962-23de-11e5-90f5-e0ccd954814b.png">
<img width="777" alt="screen shot 2015-07-06 at 12 55 04 pm" src="https://cloud.githubusercontent.com/assets/3640800/8531754/70a36584-23de-11e5-9bda-816ecaebaea9.png">
<img width="773" alt="screen shot 2015-07-06 at 12 55 26 pm" src="https://cloud.githubusercontent.com/assets/3640800/8531755/70a3c65a-23de-11e5-97fa-31efd7a57763.png">